samples/openapi3/client/petstore/go/store_api_test.go

```diff
@@ -24,7 +24,7 @@ func TestPlaceOrder(t *testing.T) {
 		// Skip parsing time error due to error in Petstore Test Server
 		// https://github.com/OpenAPITools/openapi-generator/issues/1292
 		if regexp.
-			MustCompile(`^parsing time.+cannot parse "\+0000"" as "Z07:00"$`).
+			MustCompile(`as "Z07:00"$`).
```

Comment:
This change matches a change that was [made in November 2022 in `samples/client/petstore/go/store_api_test.go`](https://github.com/OpenAPITools/openapi-generator/commit/4487042f0d4ef1f4686950f0c1ddac6fe6d44f98#diff-28b188fcd5f858971f198fe12c30c5e3b728bf5a8e327f385c99f2f5dd936ceaR27).

I wouldn't expect my template change to impact this test...was this broken before or did my template change break it?

Reply:
It was working before. To run the tests locally, please refer to https://github.com/OpenAPITools/openapi-generator/wiki/Integration-Tests#how-to-add-integration-tests-for-new-petstore-samples

Reply:
I did run the tests locally to fix the issue but hadn't run them against `master` locally to confirm the initial state.  It took me a minute to connect the new test failure to my change because I didn't change anything about time parsing.  The test was passing before because errors weren't being returned at all, so there was no time parse error to ignore; with my change to return errors, the test started failing because the regex it was using to identify time parse errors was wrong.

Reply:
@ctreatma thanks for the details. Can you please PM me vis Slack when you've time?

https://join.slack.com/t/openapi-generator/shared_invite/zt-12jxxd7p2-XUeQM~4pzsU9x~eGLQqX2g